core/kazoo_bindings/src/kazoo_bindings.erl

```diff
@@ -819,13 +818,14 @@ bindings(Routing) ->
                              }=Bind, Acc) ->
                       case Binding =:= Routing
                           orelse matches(BParts, RoutingParts)
-                          orelse matches(RoutingParts, BParts)
```

Comment:
better..., change bindings to work like `fold`and `map `. bindings/0 use rt_options, bindings/1 should take Options. `matches`above should be replaced with `kazoo_bindings_rt:matches(Options, BParts, RoutingParts)`